.github/settings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "repos": {
+    "description": "A Terraform module that manages the network resources from the azurerm provider.",
+    "homepage": "https://telekom-mms.github.io",
```

Comment:
This does not exist and I'm not sure if anybody will ever create it. Is this used anywhere except on Github? If not, let's just omit it.

Reply:
solved by https://github.com/telekom-mms/terraform-azurerm-network/pull/23/commits/eddf7ca51702c34692f101f8178c9d4c0b7609fe